protocols/rendezvous/src/client.rs

```diff
@@ -295,6 +295,7 @@ impl NetworkBehaviour for Behaviour {
 
                     return Poll::Ready(new_to_swarm);
                 }
+                Poll::Ready(_) => return Poll::Pending,
```

Comment:
Instead of adding this here, we need to extend the branch above that calls `map_out`.